components/dropdowns.py

```diff
@@ -111,182 +104,182 @@ def chart_selection(
 }
 
 pmv_en_humidity_selection = {
-    "id": ElementsIDs.PMV_EN_HUMIDITY_SELECTION.value,
+    "id": ElementsIDs.HUMIDITY_SELECTION.value,
     "question": "",
     "options": [
-        ModelInputsSelectionHumidityPmvEN16798.h_1.value,
-        ModelInputsSelectionHumidityPmvEN16798.h_2.value,
-        ModelInputsSelectionHumidityPmvEN16798.h_3.value,
-        ModelInputsSelectionHumidityPmvEN16798.h_4.value,
-        ModelInputsSelectionHumidityPmvEN16798.h_5.value,
+        HumiditySelection.h_1.value,
```

Comment:
h_1 has no meaning, use a more descriptive attribute

Reply:
get it